### ISSUE TITLE
2.3.10 Installing each time machine starts up

### ISSUE THREAD

**Issue by apwestgarth** [AutomatedArchitecture/SirenOfShame#71]
Hi,
    I have two machines set to update automatically, however each time they boot up SirenOfShame is updating to 2.3.10 despite having both checked both machines and the client reporting it is up to date :S  

Any ideas on how to resolve?

**Comment by lprichar** [comment]
That's a first. What version are they actually running? 2.3.10 or 2.4.4?

**Comment by apwestgarth** [comment]
2.3.10

**Comment by lprichar** [comment]
Wow, that's really annoying. Can't imagine why no one's reported this until now, unless everyone's on 2.4.X. Upgrading might be a quick fix for you, regardless I'll look into the underlying issue.

**Comment by lprichar** [comment]
tl;dr: Problem should be solved. Please try installing 2.3.10.2 from http://sirenofshame.com/downloads. 

Optional details: I should have read the wyBuild documentation more closely. It says quite clearly you need to use non-advertised shortcuts, which the startup shortcut was not.  That's not something I can easily fix with another wyBuild update, so unfortunately it needs to go out as a new msi.  The good news is it'll be fixed for everyone going forward, so thanks for taking the time to report it.

**Comment by lprichar** [comment]
I suspect this solution will work for you, please re-open if it doesn't.

**Comment by apwestgarth** [comment]
Yes that resolved it thanks! :+1:
